autofix/ai.go

```diff
@@ -0,0 +1,126 @@
+package autofix
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/google/generative-ai-go/genai"
+	"google.golang.org/api/option"
+
+	"github.com/securego/gosec/v2/issue"
+)
+
+const (
+	GeminiModel    = "gemini-1.5-flash"
+	AIPrompt       = "In golang, what is the solution to fix the error \"%s\". Answer limited to 200 words"
```

Comment:
```suggestion
	AIPrompt       = "In Go, what is the solution to fix the error %q. Answer limited to 200 words"
```

The language name is Go. Golang was the DNS of golang.org where Go was promoted.

Golang is also useful when using search engine. But here, Go is enough

Also, if you want to quote something, use %q it's its purpose

Alternatively, you can use string literal


```suggestion
	AIPrompt       = `In Go, what is the solution to fix the error "%s". Answer limited to 200 words`
```

But, I would recommend using %q anyway